plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -104,14 +121,22 @@ export const AboutContent = ({ entity }: Props) => {
           />
         </AboutField>
       )}
-      {!isSystem && !isDomain && (
+      {(isAPI ||
+        isComponent ||
+        isResource ||
+        isTemplate ||
+        isGroup ||
+        isLocation ||
```

Comment:
Considering that `type` and `location` are required fields and missing values shouldn't be allowed by the catalog validation, we could simply change it to "display if available".

The most value of this "No ..." fallback is in the optional fields "system", "domain", and "tags" to show that this feature is available and might be recommended. Even though a user still doesn't receive any help on how to set these values. An option could be to add a hover text that says what to do in this case.

Depends on how opinionated we want to be about the data model in the default `AboutCard`.

Reply:
Yep if we want to add even more help text and special behaviors then it might perhaps be worth doing it your way.

The main thing I have against it, is that it makes it less usable for custom kinds. We expect companies to expand the model with their own kinds, and it's maybe convenient to be able to use the unmodified about box for them too. Honestly, this is kinda arguable, another way to look at it is that the superior approach is to make it very easy to compose a custom about box in any way you like.

In summary I don't have a strong opinion here :) The only thing that held back my approve was actually the question about Link targets above.

Reply:
The logic here is that it now always shows the field if it is available and hide them if not. But for certain selected kinds,  it also shows whether a field is _not_ available. But this is now a positive check and not a negative check that falls back to show every field for any unrecognized entity kind.

Lets go with this for now and see if someone has more ideas in the future :-)